test/test_feedback.h

```diff
@@ -0,0 +1,22 @@
+/* Header for feedback unit tests
+ * Provides mock helpers, externs and prototypes used by
+ * `test/test_feedback.c`.
+ */
+
+#ifndef TEST_TEST_FEEDBACK_H
```

Comment:
Why the `TEST_TEST` prefix?
```suggestion
#ifndef TEST_FEEDBACK_H
```